.github/workflows/fedora_35.yml

```diff
@@ -0,0 +1,69 @@
+name: fedora_35
```

Comment:
Please, add a [changelog entry](https://github.com/tarantool/tarantool/blob/master/doc/changelogs.md) (see other entries in the `changelogs/unreleased` directory regarding enabling Linux distributions support for example).

Reply:
Done